commerce_coordinator/apps/lms/views.py

```diff
@@ -95,20 +74,18 @@ def _redirect_response_payment(self, request):
         return redirect
 
     @staticmethod
-    def _add_utm_and_waffle_flags_params_to_url(url, params):
+    def _add_utm_params_to_url(url, params):
```

Comment:
This function is just adding the utm params which isn't what we intend to do.
We should extract the query params that were sent to coordinator with our request and append those query params to our redirect url. You can make a new utility for this purpose(`_add_query_params_to_redirect_url`).

Reply:
I'm extracting the query params from the request object in the function above and passing it in as the argument 'params' in this function.

Reply:
The argument 'params' is a list of all query params extracted from the request url